src/main/kotlin/no/nav/dagpenger/inntekt/db/ArenaMappingMigrator.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.dagpenger.inntekt.db
+
+import javax.sql.DataSource
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import kotliquery.using
+import mu.KotlinLogging
+import org.intellij.lang.annotations.Language
+
+internal class ArenaMappingMigrator(private val datasource: DataSource) {
+
+    companion object {
+        private val logger = KotlinLogging.logger {}
+    }
+
+    @Language("sql")
+    private val statement: String = """
+                  INSERT INTO inntekt_v1_person_mapping (inntektid, aktørid, vedtakid, beregningsdato)
+                  SELECT inntektid, aktørid, vedtakid::text, beregningsdato
+                  FROM inntekt_v1_arena_mapping 
+                  WHERE inntektid NOT IN (SELECT inntektid from inntekt_v1_person_mapping) ON CONFLICT DO NOTHING 
```

Comment:
Nit: Er ikke dette smør på flesk(antar at inntektid er unik). Lurer også på om det er verdt å 

1. Ta ned appen. 
2. Sette antall replikas til 1. 
3. Gjøre migrering som en del av flyway

Dette for å sikre at ingen kan skrive til inntekt_v1_arena_mapping migrering pågår.

Reply:
Enig, trenger ikke ON CONFLICT i alle fall

Det skrives ikke lenger til `inntekt_v1_arena_mapping` skriving til den ble tatt bort i  https://github.com/navikt/dp-inntekt-api/pull/88 

Vurderte å ta via flyway men ville logge litt og ha kontroll. 

Jeg vurderer om jeg skal legge til en  [advisory lock](https://hashrocket.com/blog/posts/advisory-locks-in-postgres) for å la en instans gjøre migreringen.